src/lava/proc/io/bridge/extractor.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import typing as ty
+
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.ports.ports import InPort
+
+from lava.magma.core.resources import CPU
+from lava.magma.core.decorator import implements, requires
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.model.py.ports import PyOutPort, PyInPort
+
+from lava.magma.compiler.channels.pypychannel import PyPyChannel
+from lava.magma.runtime.message_infrastructure.multiprocessing import \
+    MultiProcessing
+
+from lava.proc.io.bridge.utils import ChannelConfig, ChannelSendBufferFull, \
+    ChannelRecvBufferEmpty, ChannelRecvBufferNotEmpty,\
+    validate_shape, validate_buffer_size, validate_channel_config, \
+    send_data_blocking, send_data_non_blocking_drop, \
+    recv_empty_blocking, recv_empty_non_blocking_zeros, recv_not_empty_fifo, \
+    recv_not_empty_accumulate
+
+
+class Extractor(AbstractProcess):
+    """Process that gets data out of Lava.
```

Comment:
You can cut this first line of the docstring and the resulting text will be shorter and more accurate.

Reply:
Removed this line.